tests/init.spec.js

```diff
@@ -0,0 +1,234 @@
+const assert = require('assert');
+const { execSync } = require('child_process');
+const { existsSync, mkdtempSync, readFileSync, rmdirSync } = require('fs');
+const { parse, resolve } = require('path');
+// Path to cli relative to tmp working directory.
+const cli = '../../../cli.js';
+// Create tmp directory.
+const cwd = mkdtempSync(resolve(__dirname, 'tmp/init-'));
+const initCmd = `${ cli } init`;
+const stdio = 'pipe';
+const expectedPackageCommands = {
+  calliope: 'yarn install && calliope',
+  build: 'yarn calliope build',
+  start: 'yarn calliope start',
+  lint: 'yarn calliope lint',
+  test: 'yarn lint',
+};
+const files = {
+  config: 'calliope.config.js',
+  env: '.env-sample',
+  eslint: '.eslintrc.yml',
+  gitignore: '.gitignore',
+  readme: 'README.md',
+  stylelint: '.stylelintrc.yml',
+};
+// Total number of possible errors is equal to the number of boilerplates
+// available to copy plus one error to account for non-existent manifest error.
+const maxErrors = Object.keys(files).length + 1;
+
+/**
+ * HAPPY PATH.
```

Comment:
👏🏼 for adding tests.

Did you mean to leave this? 👀

Reply:
Yes, I did. Tests were not part of the original scope, so I wrote very rudimentary tests without any sort of test-running framework, i.e. this is quick & dirty work. It's all a bit too imperative at the moment and, in my opinion, difficult to follow. So I wrote these big, ALLCAPS headings to find my way around this file and tons of comments to make sense of each step.

After sleeping on this some, I think it's worth attempting to refactor tests to be more declarative and to use a test-running framework within this PR. I'll do that this morning. That shouldn't take too long and will help make tests easier to grok and navigate. If it turns out that takes longer than a couple of hours, I'll create a ticket to do all of that una follow-up PR.

Reply:
I went and refactored tests to use `mocha` as a test-runner. While I was in there, I extracted common assertions into reusable functions and added iterators to run assertions for each individual file where appropriate. I think tests are far more robust now.